src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/AllMetrics.java

```diff
@@ -1316,6 +1317,79 @@ public String toString() {
     }
   }
 
+  public enum ShardIndexingPressureNodeRoleType {
+    COORDINATING(Constants.COORDINATING_VALUE),
+    PRIMARY(Constants.PRIMARY_VALUE),
+    REPLICA(Constants.REPLICA_VALUE);
```

Comment:
It's a deviation from the ES nodes roles enlisted [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#node-roles). It will be confusing with people.